test/cmdlineTests/asm_json_import_difficulty_prevrandao/output

```diff
@@ -1,6 +1,5 @@
 Opcodes:
 PREVRANDAO PREVRANDAO 
 EVM assembly:
-    /*   */
```

Comment:
Why is this now gone?

Reply:
Because source location `0:0` is a valid location, but due to the bug it should ne `-1:-1`. these comments are only generated for valid source locations, where they consists of the code snippet at these source locations.